src/generate.ts

```diff
@@ -34,6 +40,7 @@ const sqlTypeToTypescriptType = (
 	m: RequiredModelSubset,
 	f: AbstractSqlField,
 	mode: Mode,
+	opts?: Options,
```

Comment:
I think the `opts` argument should be required on all functions except the exported abstractSqlToTypescriptTypes where it can be a default, ie `opts: Options = {},`